pkg/plugins/policies/meshfaultinjection/plugin/v1alpha1/plugin_test.go

```diff
@@ -116,32 +116,61 @@ var _ = Describe("MeshFaultInjection", func() {
 			},
 			fromRules: core_rules.FromRules{
 				Rules: map[core_rules.InboundListener]core_rules.Rules{
-					{Address: "127.0.0.1", Port: 17777}: {{
-						Subset: core_rules.Subset{
-							{
-								Key:   "kuma.io/service",
-								Value: "demo-client",
+					{Address: "127.0.0.1", Port: 17777}: {
+						{
+							Subset: core_rules.Subset{
+								{
+									Key:   "kuma.io/service",
+									Value: "demo-client",
```

Comment:
so this is not targeted because the name of this service is `backend` right?

Reply:
It's inbound policy, so MeshFaultInjection will configure faults on `backend`s listener for the traffic comming from `demo-client` I believe.